.jenkins/pytorch/win-test.sh

```diff
@@ -64,6 +64,10 @@ run_tests() {
 }
 
 run_tests && assert_git_not_dirty && echo "TEST PASSED"
+if [[ $? -gt 0 ]]; then
```

Comment:
Are you sure the exit code will be greater than zero when the test is failing?

Reply:
Also it sounds weird to print out `TEST PASSED` even though the pass is failing.

Reply:
BTW, why `set -e` in line1 doesn't catch errors in this case?

Reply:
> Also it sounds weird to print out `TEST PASSED` even though the tests are failing.

"&&" is used to chain commands together, such that the next command is run if and only if the preceding command exited without errors

Reply:
> Are you sure the exit code will be greater than zero when the test is failing?

according to https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html, Exit statuses fall between 0 and 255. Maybe not equal zero is more safe, in case some one write exit /b -1

Reply:
> BTW, why `#!/bin/bash -ex` in line1 doesn't catch errors in this case?
> Updated answer:
> According to the local testing results, the shebangs won't work but calling `set -ex` will work.
> So actually, would `set -ex` be a better solution than checking the exit code here?

according to https://unix.stackexchange.com/questions/172541/why-does-exit-1-not-exit-the-script
"() runs commands in the subshell, so by exit you are exiting from subshell and returning to the parent shell."

Reply:
> BTW, why `#!/bin/bash -ex` in line1 doesn't catch errors in this case?
> Updated answer:
> According to the local testing results, the shebangs won't work but calling `set -ex` will work.
> So actually, would `set -ex` be a better solution than checking the exit code here?

according to https://unix.stackexchange.com/questions/172541/why-does-exit-1-not-exit-the-script
"() runs commands in the subshell, so by exit you are exiting from subshell and returning to the parent shell."

Reply:
Why don't we just use `set -ex`? It is a lot better than manually check the exit code.

Reply:
> > Also it sounds weird to print out `TEST PASSED` even though the tests are failing.
> 
> "&&" is used to chain commands together, such that the next command is run if and only if the preceding command exited without errors

Sorry, didn't see that carefully. You are right.